domain/consensus/utils/consensusserialization/utxo.go

```diff
@@ -0,0 +1,134 @@
+package consensusserialization
+
+import (
+	"bytes"
+	"encoding/binary"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/transactionid"
+	"io"
+
+	"github.com/pkg/errors"
+
+	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+)
+
+// outpointIndexByteOrder is the byte order for serializing the outpoint index.
+// It uses big endian to ensure that when outpoint is used as database key, the
+// keys will be iterated in an ascending order by the outpoint index.
```

Comment:
This comment speaks about database serialization. Please remove it and make the byte order be little endian.